packages/vscode-extension/package.nls.json

```diff
@@ -278,12 +278,13 @@
   "teamstoolkit.handlers.installOfficeAddinDependencyCancelled": "Dependency installation is canceled, but you can install dependencies manually by clicking the 'Development - Check and Install Dependencies' button on the left side.",
   "teamstoolkit.localDebug.learnMore": "Get More Info",
   "teamstoolkit.localDebug.m365TenantHintMessage": "After enrolling your developer tenant in Office 365 Target Release, enrollment may come into effect in couple of days. Click 'Get More Info' button for details on setting up dev environment to extend Teams apps across Microsoft 365.",
-  "teamstoolkit.handlers.askInstallCopilot": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps, you need to install both GitHub Copilot and GitHub Copilot Extension for Teams Toolkit (@teamsapp).",
-  "_teamstoolkit.handlers.askInstallCopilot.comment": "@teamsapp is used to call Teams agent in Github Copilot Chat and should not be translated.",
+  "teamstoolkit.handlers.askInstallCopilot": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps or customizing Microsoft 365 Copilot, you need to install GitHub Copilot first.",
   "teamstoolkit.handlers.askInstallCopilot.install": "Install GitHub Copilot",
-  "teamstoolkit.handlers.askInstallCopilot.installTeamsApp": "Install @teamsapp",
-  "_teamstoolkit.handlers.askInstallCopilot.installTeamsApp.comment": "@teamsapp is used to call Teams agent in Github Copilot Chat and should not be translated.",
-  "teamstoolkit.handlers.installExtension.output": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps, you need to install both GitHub Copilot Chat following \"%s\" and GitHub Copilot Extension for Teams Toolkit (@teamsapp) following \"%s\".",
+  "teamstoolkit.handlers.askInstallTeamsAgent": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps or customizing Microsoft 365 Copilot, you need to install GitHub Copilot Extension for Teams Toolkit. If you've already installed it, confirm.",
```

Comment:
Proposed: To use GitHub Copilot Extension for Teams Toolkit, please install it first. If you’ve already installed it, confirm below.
Logic: We can skip "when developing Teams apps or customizing Microsoft 365 Copilot" because the context of the notification already implies the purpose of the installation.